charts/mlflow/values.yaml

```diff
@@ -242,3 +242,19 @@ extraVolumeMounts:
 #   mountPath: /mlflow/data
 # - name: example
 #   mountPath: /example
+
+livenessProbe:
+  enabled: true
+  healthCheckPath: "/"
+  initialDelaySeconds: 30
+  periodSeconds: 20
+  timeoutSeconds: 6
```

Comment:
These are must be Kubernetes' default values. You can find them [from here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes).